windows_containers/windows-containers-release-notes-10-16-x.adoc

```diff
@@ -0,0 +1,101 @@
+
+[id="wmco-10-16-0-network-charts"]
+==== Pod network metrics now display for the pods on Windows nodes
+
+The *Network in* and *Network out* charts are now available for Windows pods on the *Pod details* page in the {product-title} web console. You can xref:../observability/monitoring/managing-metrics.adoc#about-querying-metrics_managing-metrics[query the metrics] by running PromQL queries. The charts previously reported *No datapoints found*.
+
+[id="wmco-10-16-0-cpu-memory"]
+==== Pod CPU and memory metrics now display for the pods on Windows nodes
+
+The CPU and memory usage metrics are now available for Windows pods on the *Pods* and *Pod details* pages in the {product-title} web console. You can xref:../observability/monitoring/managing-metrics.adoc#about-querying-metrics_managing-metrics[query the metrics] by running PromQL queries. The chart previously reported *No datapoints found*.
+
+[id="wmco-10-16-0-operator-kube"]
+==== Kubernetes upgrade
+
+The WMCO now uses Kubernetes 1.29.
+
+[id="wmco-10-16-0-bug-fixes"]
+=== Bug fixes
+
+Because the WICD service account was missing a required secret, the WMCO was unable to properly configure Windows nodes in a Nutanix cluster. With this fix, the WMCO creates a long-lived token secret for the WICD service account. As a result, the WMCO is able to configure a Windows node on Nutanix. (link:https://issues.redhat.com/browse/OCPBUGS-22680[*OCPBUGS-22680*])
+
+Previously, the WMCO performed a sanitization step that incorrectly replaced commas with semicolons in the user's cluster-wide proxy configuration. This caused Windows to ignore the values set in the `noProxy` environment variable. As a consequence, the WMCO incorrectly sent traffic through the proxy for the endpoints specified in the `no-proxy` parameter. With this fix, the sanitization step that replaced commas with semicolons was removed. As a result, web requests from a Windows node to a cluster-internal endpoint or an endpoint that exists in the `no-proxy` parameter does not go through the proxy. (link:https://issues.redhat.com/browse/OCPBUGS-24264[*OCPBUGS-24264*])
+
+// Copied from 4.15
+Previously, because of bad logic in the networking configuration script, the WMCO was incorrectly reading carriage returns in the containderd CNI configuration file as changes, and identified the file as modified. This caused the CNI configuration to be unnecessarily reloaded, potentially resulting in container restarts and brief network outages. With this fix, the WMCO now reloads the CNI configuration only when the CNI configuration is actually modified. (link:https://issues.redhat.com/browse/OCPBUGS-2887[*OCPBUGS-2887*])
```

Comment:
```suggestion
Previously, because of bad logic in the networking configuration script, the WMCO was incorrectly reading carriage returns in the containderd CNI configuration file as changes, and identified the file as modified. This behavior caused the CNI configuration to be unnecessarily reloaded, potentially resulting in container restarts and brief network outages. With this fix, the WMCO now reloads the CNI configuration only when the CNI configuration is actually modified. (link:https://issues.redhat.com/browse/OCPBUGS-2887[*OCPBUGS-2887*])
```
- [Per ISG guidance](https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references), make sure pronoun references are clear